spec/models/groups_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  it 'is valid with valid attributes' do
+    user = User.create(name: 'Javier', email: 'email@gmail.com', password: 'password')
+    group = Group.new(user:, name: 'Example', icon: 'example.png')
+    expect(group).to be_valid
+  end
```

Comment:
- [ ] Good job adding model and controller specs, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=integration%20tests%20for%20all%20the%20most%20important%20components%20of%20your%20RoR%20application.), Please also add integration(features) specs for used app `views`.